manual/cops_rspec.md

```diff
@@ -241,16 +241,17 @@ Enabled | No
 
 `context` block descriptions should start with 'when', or 'with'.
 
-'without'
+'with', 'without', and 'if'
   Prefixes:
+    - for
     - when
     - with
     - without
     - if
 
 ### Examples
 
-#### `Prefixes` configuration option, defaults: 'when', 'with', and
+#### `Prefixes` configuration option, defaults: 'for', 'when',
```

Comment:
Looks like there's a problem with the existing documentation here, I've opened #758